pkgs/os-specific/linux/xf86-input-cmt/default.nix

```diff
@@ -30,7 +30,7 @@ stdenv.mkDerivation rec {
     description = "Chromebook touchpad driver.";
     license = licenses.bsd3;
     platforms = platforms.linux;
-    homepage = "www.github.com/hugegreenbug/xf86-input-cmt";
+    homepage = "https://www.github.com/hugegreenbug/xf86-input-cmt";
```

Comment:
Could take `homepage` from `src.meta` here.

Reply:
What do you mean? Does `fetchFromGitHub` generate a URL?

Reply:
Yes, but I prefer not using it so that I can quickly visit the homepage from the source code.